dev-perl/compress_bzip2/compress_bzip2-2.28.recipe

```diff
@@ -0,0 +1,41 @@
+SUMMARY="Interface to Bzip2 compression library"
+DESCRIPTION="This module provides a Compress::Zlib like Perl interface to the bzip2 library. It \
+uses the low level interface to the bzip2 algorithm, and reimplements all high level routines."
+HOMEPAGE="https://metacpan.org/pod/Compress::Bzip2"
+COPYRIGHT="1999-2020 Rob Janes"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://cpan.metacpan.org/authors/id/R/RU/RURBAN/Compress-Bzip2-$portVersion.tar.gz"
+CHECKSUM_SHA256="859f835c3f5c998810d8b2a6f9e282ff99d6cb66ccfa55cae7e66dafb035116e"
+SOURCE_DIR="Compress-Bzip2-$portVersion"
+
+ARCHITECTURES="any"
+
+PROVIDES="
+	compress_bzip2 = $portVersion
+	"
+REQUIRES="
+	haiku
+	vendor_perl
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	devel:libbz2
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc
```

Comment:
If this requires gcc, doesn't this contain binary code then? It shouldn't use "any" as architecture in this case, but rather "all" and SECONDARY_ARCHITECTURES="x86" (to provide both versions for x86_gcc2).

Reply:
Had been tinkering about that too yesterday, maybe it was getting late? :)
Without it it fails:
```
make[1]: Entering directory '/sources/Compress-Bzip2-2.28/bzlib-src'
gcc -c   -fwrapv -fno-strict-aliasing -pipe -D_FORTIFY_SOURCE=2 -O   -DVERSION=\"\" -DXS_VERSION=\"\" -fpic "-I/packages/perl-5.32.1-1/.self/lib/perl5/5.32.1/x86_64-haiku/CORE"   blocksort.c
/bin/sh: line 1: gcc: command not found
```
I'll change this (and pathtools) to use "all" instead of "any" 👍